shared/src/validation-schemas/user/user-sign-in.validation-schema.ts

```diff
@@ -0,0 +1,25 @@
+import * as Joi from 'joi';
+import { getNameOf } from '~/helpers/helpers';
+import { UserSignInRequestDto } from '~/common/types/types';
+import { UserValidationMessage } from '~/common/enums/enums';
+import { UserSignInValidationRules } from '~/common/enums/enums';
```

Comment:
Please pay attention on imports that were already made
```suggestion
import { UserSignInValidationRules, UserValidationMessage } from '~/common/enums/enums';
```